opm/models/utils/simulator.hh

```diff
@@ -172,6 +172,18 @@ public:
         checkParallelException("Allocating the simulation vanguard failed: ",
                                exceptionThrown, what);
 
+        // Only relevant for CpGrid
+        if (verbose_)
+            std::cout << "Adding LGRs, if any\n" << std::flush;
```

Comment:
Please do not use cout, but logging.
Also it might make sense to move the message into addLgrs